src/otable/orow.py

```diff
@@ -1,7 +1,8 @@
 """Table data types backed by objects."""
 
 import collections.abc
-from typing import Iterable, Sequence, TypeVar, cast, overload
+from .ocolumn import OColumn
+from typing import Any, Iterable, Sequence, TypeVar, cast, overload
 
 T = TypeVar('T')
```

Comment:
Is it guaranteed that all members of a row have the same type? (Essentially, this means all members of a table must have the same type.)